cScripts/functions/vehicle/fn_vehicle_applyTextures.sqf

```diff
@@ -0,0 +1,28 @@
+#include "..\script_component.hpp";
+/*
+ * Author: SPC.Turn.J
+ * This function applies textures to a given vehicle.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Textures <ARRAY>
+ * 0: Start Index <INT>
+ * Example:
+ * ["vic", "_myTextures"] call cScripts_fnc_vehicle_applyTextures;
+ * ["vic", "_myTextures", 3] call cScripts_fnc_vehicle_applyTextures;
+ */
+
+params ["_vehicle", "_textureArray", ["_startInd",1]];
+
+private _texturesArrLastIndex = count _textureArray;
+private _texturePosition = [];
+
+for "_i" from _startInd to _texturesArrLastIndex do {
+    _ind = _i - 1;
+    _texturePosition pushBack _ind;
+};
+
+private _texturesHash = _texturePosition createHashMapFromArray _textureArray;
+{
+    _vehicle setObjectTextureGlobal [_x, _y];
+} forEach _texturesHash;
```

Comment:
With give array exsample form above this could be simplyfied.

```suggestion
#include "..\script_component.hpp";
/*
 * Author: SPC.Turn.J
 * This function applies textures to a given vehicle.
 *
 * Arguments:
 * 0: Vehicle <OBJECT>
 * 1: Textures <ARRAY>
 *
 * Example:
 * ["vic", [[1, "path/to/texture.paa"]]] call cScripts_fnc_vehicle_applyTextures;
 * ["vic",] call cScripts_fnc_vehicle_applyTextures;
 *
 * Public: No
 */

params [
    ["_vehicle", objNull [objNull]],
    ["_textureIndexMap", [], [[]]]
];

if (count _textureIndexMap == 0) exitWith {};

private _map = createHashMapFromArray _textureIndexMap;

{
    _vehicle setObjectTextureGlobal [_x, _y];
} forEach _map;
```